### PR TITLE
Release 0.19.8

### PR DESCRIPTION
**Pull request by noklam**
Signed-off-by: Nok <nok.lam.chan@quantumblack.com>

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -7,6 +15,7 @@
 * Moved `_find_run_command()` and `_find_run_command_in_plugins()` from `__main__.py` in the project template to the framework itself.
 * Fixed a bug where `%load_node` breaks with multi-lines import statements.
 * Fixed a regression where `rich` mark up logs stop showing since 0.19.7.
+
```
Comment (merelcht) [inline RELEASE.md:18]: Can you add that we fixed the regression for the stacktrace not showing anymore?

Comment (noklam) [reply RELEASE.md:18]: Done

File: docs/source/development/commands_reference.md [hunk 2]
```diff
@@ -109,14 +109,14 @@ Returns output similar to the following, depending on the version of Kedro used
...
-v0.19.7
+v0.19.8
 
 Kedro is a Python framework for
 creating reproducible, maintainable
 and modular data science code.
 
 Installed plugins:
-kedro_viz: 4.4.0 (hooks:global,line_magic)
+kedro_viz: 9.2.0 (hooks:global,line_magic)
```
Comment (merelcht) [inline docs/source/development/commands_reference.md:119]: Oh wow have we just neglected updating this?! 👀

**Review by merelcht — APPROVED** [review]
Let's ship it! 🎉

File: docs/source/development/commands_reference.md (hunk 2 above)
Comment (noklam) [reply docs/source/development/commands_reference.md:119]: @merelcht Yeah I don't think we have the habit to make sure this is up to date, but this is surely quite outdated🤓

File: docs/source/robots.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
 User-agent: *
```
Comment (astrojuanlu) [inline docs/source/robots.txt:1]: Can we actually remove this file? https://github.com/kedro-org/kedro/pull/4055

File: RELEASE.md [hunk 4]
```diff
@@ -1,12 +1,22 @@
...
 
 ## Bug fixes and other changes
+* Fixed a bug that caused tracebacks disappeared from CLI runs.
 * Moved `_find_run_command()` and `_find_run_command_in_plugins()` from `__main__.py` in the project template to the framework itself.
 * Fixed a bug where `%load_node` breaks with multi-lines import statements.
 * Fixed a regression where `rich` mark up logs stop showing since 0.19.7.
+
 ## Breaking changes to the API
 
 ## Documentation changes
```
Comment (astrojuanlu) [inline RELEASE.md:22]: Missing a few community contributions: cclauss #4102 #4099, eltociear #4101, ltalirz #4089

Comment (noklam) [reply RELEASE.md:22]: See [add community contributions](https://github.com/kedro-org/kedro/pull/4107/commits/5757b48e26fefd5cec5b13037949183c929e7ba8)

**Review by astrojuanlu — APPROVED** [review]
🚢

**Pull request merged**